### PR TITLE
chore(deps): override package_info_plus to allow web 0.5.0

### PR DESCRIPTION
**Pull request by Leptopoda**
The new package_info_plus is not going to be released soon [comment](https://github.com/fluttercommunity/plus_plugins/pull/2605#issuecomment-1954955437).
Let's override it for now to unblock:
#1609 
#1607
#1627

### CODE REVIEW

File: packages/neon_framework/pubspec.yaml [hunk 1]
```diff
@@ -88,3 +88,10 @@ flutter:
       web:
         pluginClass: WebNeonPlatform
         fileName: src/platform/web.dart
+
+dependency_overrides:
+  package_info_plus:
+    git:
+      url: https://github.com/fluttercommunity/plus_plugins
+      path: packages/package_info_plus/package_info_plus
+      ref: baa336fe79c29411542fc343d332846bcd7752c1
```
Comment (provokateurin) [inline packages/neon_framework/pubspec.yaml:97]: This should appear in the pubspec.lock

Comment (Leptopoda) [reply packages/neon_framework/pubspec.yaml:97]: ok `wakelock_plus` a dependency of `neon_news` was also depending on `package_info_plus`.
For now I just added the second override to the app itself as neon_news does not depend on web in any way.

If you want I can move it to news but I fear we forget to remove it from there afterwards.

File: packages/app/pubspec_overrides.yaml [hunk 2]
```diff
@@ -20,5 +20,10 @@ dependency_overrides:
     path: ../neon/neon_notifications
   nextcloud:
     path: ../nextcloud
+  package_info_plus:
+    git:
+      url: https://github.com/fluttercommunity/plus_plugins
+      ref: baa336fe79c29411542fc343d332846bcd7752c1
+      path: packages/package_info_plus/package_info_plus
   sort_box:
     path: ../sort_box
```
Comment (provokateurin) [inline packages/app/pubspec_overrides.yaml:29]: When I run `melos clean && melos bs` sort_box is before the new override

Comment (Leptopoda) [reply packages/app/pubspec_overrides.yaml]: melos clean was not deleting `packages/app/pubspec_overrides.yaml` for me. After a manual rm and a `melos bs` it works.
Strange

**Review by provokateurin — APPROVED** [review]

**Pull request merged**